backend/user-service/src/main/java/com/speakapp/userservice/controllers/UserController.java

```diff
@@ -22,4 +25,38 @@ public AppUserDTO getUserById(@PathVariable(name = "userId") UUID userId)
             throws UserNotFoundException {
         return userService.getUser(userId);
     }
+
+    @PostMapping("")
+    @ResponseStatus(HttpStatus.CREATED)
+    public void createUser(@RequestBody AppUserCreateDTO userCreateDTO) {
+        userService.createUser(userCreateDTO);
+    }
+
+    @PutMapping
+    @ResponseStatus(HttpStatus.CREATED)
```

Comment:
Zamiast Created dałbym response status 200 (bez annotacji), bo created sugeruje jakby nowy user się tworzył, natomiast w przypadku updatu zdjęć to może to mieć rzeczywiście sens (bo będzie robił się jakiś tam zasób w blob storage)